cypress/integration/ModelPage_spec.js

```diff
@@ -1,32 +1,32 @@
 describe('/state/:id', () => {
   it(`Stay At Home has the correct title & description`, () => {
```

Comment:
These expectation names are now incorrect. Probably should rename this one to match reality, and delete the other two (see comment on lines 12 and 13 where someone anticipated this  eventuality).

Ultimately it'd be nice if there was a way to write tests that didn't include the whole shebang, but probably not as part of this PR :)

```suggestion
  it(`State page the correct title & description`, () => {
```